src/command_modules/azure-cli-sql/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 Unreleased
 ++++++++++++++++++++
 * az sql server list --resource-group argument is now optional. If not specified, all sql servers in the entire subscription will be returned.
+* Adding support for SQL Transparent Data Encryption (TDE) and TDE with Bring Your Own Key
```

Comment:
The other two changes listed here were in fact released as 2.0.12. Please fix this bad merge and then we'll get this in.